src/js/components/RadioButton/doc.js

```diff
@@ -68,6 +68,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '2px',
   },
+  'radioButton.check.backgroundColor': {
```

Comment:
```suggestion
  'radioButton.check.background.color': {
```
For consistency, we usually name background with simply just `background` vs `backgroundColor`. This change help us scale under the umbrella of background. so ultimately, you'll have:
```
 'radioButton.check.background': {...}
 'radioButton.check.background.color': ''

radioButton: {
   check: {
      background: {
         color: ' ',
      }
   }
}
``` 

This change may affect the rest of the PR, but it should be an easy fix.

Reply:
Thanks, I'll fix it.